warcit/warcit.py

```diff
@@ -240,7 +241,7 @@ def __init__(self, url_prefix, inputs,
             self.use_mapfile = True
             self.mapfile = mapfile
 
-        self.logfile = logfile
+        self.logfile = str(logfile + '.csv')
```

Comment:
```suggestion
        self.logfile = logfile
```
I think we actually want to put this below in the `if self.logfile:` block. Otherwise could end up with a `None.csv` logfile when we don't want/expect one.